Doc/library/types.rst

```diff
@@ -82,6 +82,32 @@ Dynamic Type Creation
 
    .. versionadded:: 3.7
 
+.. function:: get_original_bases(cls, /)
+
+    Return the objects in the bases list of the class's definition as they
+    existed prior to any modification by :meth:`~object.__mro_entries__`. This
+    is useful for introspecting :ref:`Generics <user-defined-generics>`.
+
+    Examples::
+
+        from typing import TypeVar, Generic
+
+        T = TypeVar("T")
+        class Foo(Generic[T]): ...
+        class Bar(Foo[int], float): ...
+        class Baz(list[str]): ...
+
+        Bar.__bases__ == (Foo, float)
```

Comment:
Oh, now I see. Yes, this definitely shouldn't be written like this. Either an assert or (my preferred way) some English text in a comment explaining "Now Bar.__bases__ is equal to ..."

Reply:
I think I'll stick with an assert as IMO it's easier to instantly understand

Reply:
For me personally it's harder to instantly follow than a comment written in English -- my dinosaur brain thinks of `assert` as a run-time check, not as a piece of documentation. But I know this use of `assert` is fairly common, so I accept defeat on this point.